lib/pdk/module/release.rb

```diff
@@ -95,7 +95,7 @@ def write_module_metadata!
       def default_package_filename
         return @default_tarball_filename unless @default_tarball_filename.nil?
 
-        builder = PDK::Module::Build.new(module_dir: module_path)
+        builder = Puppet::Modulebuilder::Builder.new(module_path, nil, PDK.logger)
```

Comment:
we dont care about the target dir for releases?

Reply:
Was trying to keep it as consistent with the previous code as before, no target-dir was passed to `PDK::Module::Build.new(module_dir: module_path)`. But, looks like a default was set from within the initalize method of that class, so nice catch! I'll update

Reply:
And I should have said, we aren't actually building the module here, rather retrieving the package_file name. So target-dir isn't required!